packages/terra-date-input/tests/wdio/date-input-spec.js

```diff
@@ -75,21 +75,19 @@ Terra.describeViewports('Date Input', ['medium'], () => {
   });
 
   describe('Month Select placeholder option clears value', () => {
-    it('Selects February in month select', () => {
+    it('Selects January in month select', () => {
       browser.url('/raw/tests/cerner-terra-framework-docs/date-input/day-month-year-date-input');
       Terra.hideInputCaret('input[name="terra-date-day-date-input"]');
       Terra.hideInputCaret('input[name="terra-date-year-date-input"]');
       $('select[name="terra-date-month-date-input"]').click();
       browser.keys(['ArrowDown']);
-      browser.keys(['ArrowDown']);
```

Comment:
why do we removing the arrowDown key event here !!

Reply:
The justification for removing arrowDown key event has been provided in above comment.
https://github.com/cerner/terra-framework/pull/2019#discussion_r1482512865
And after the discussion with saket its not required to get current by selecting the current month. So that i have removed the logic as well.